publish.sh

```diff
@@ -67,20 +67,18 @@ function build_clusters() {
 function push_node_base_image() {
   TARGET_IMAGE="${TARGET_REPO}/centos9:${KUBEVIRTCI_TAG}"
   echo "INFO: push $TARGET_IMAGE"
-  skopeo copy "docker-daemon:${TARGET_REPO}/centos9-base:latest" "docker://${TARGET_IMAGE}"
+  podman push $TARGET_IMAGE
```

Comment:
nit: `shellcheck` usually complains with [quote this to prevent word splitting](https://www.shellcheck.net/wiki/SC2046)